CHANGES.rst

```diff
@@ -2,10 +2,12 @@
  CHANGES
 =========
 
-4.3.1 (unreleased)
-==================
+4.4 (unreleased)
+================
```

Comment:
```suggestion
4.4.0 (unreleased)
==================
```

For consistency with all the previous releases.

Reply:
It seems my configuration of zest.releases is not so optimal for zopefoundation projects.